### PR TITLE
3/30 testing

### PR DESCRIPTION
**Pull request by gosprogrammers**
pj said to commit & push this

### CODE REVIEW

File: y2024/Crescendo/src/main/java/com/gos/crescendo2024/AprilTagCamera.java [hunk 1]
```diff
@@ -113,6 +115,7 @@ public Matrix<N3, N1> getEstimationStdDevs(Pose2d estimatedPose) {
         List<PhotonTrackedTarget> targets = m_lastPipelineResult.getTargets();
         m_numTargetsSeen = 0;
         double sumDist = 0;
+        double sumAbiguity = 0;
```
Comment (pjreiniger-aurora) [inline y2024/Crescendo/src/main/java/com/gos/crescendo2024/AprilTagCamera.java:118]: PJ can't spell

sumAmbiguity

File: y2024/Crescendo/src/main/java/com/gos/crescendo2024/RobotContainer.java [hunk 2]
```diff
@@ -241,6 +241,7 @@ private void createTestCommands() {
         shuffleboardTab.add("Auto-Intake Piece", CombinedCommands.intakePieceCommand(m_armPivotSubsystem, m_intakeSubsystem));
         shuffleboardTab.add("Auto-Shoot in Amp", CombinedCommands.ampShooterCommand(m_armPivotSubsystem, m_shooterSubsystem, m_intakeSubsystem));
         shuffleboardTab.add("Full Field Feed", CombinedCommands.feedPieceAcrossFieldWithVision(m_driverController, m_chassisSubsystem, m_armPivotSubsystem, m_shooterSubsystem, m_intakeSubsystem));
+        shuffleboardTab.add("Prepate Tunable Shot", CombinedCommands.prepareTunableShot(m_armPivotSubsystem, m_shooterSubsystem));
```
Comment (pjreiniger-aurora) [inline y2024/Crescendo/src/main/java/com/gos/crescendo2024/RobotContainer.java:244]: PJ can't spell. Prepare

File: y2024/Crescendo/src/main/java/com/gos/crescendo2024/auton/Autos.java [hunk 3]
```diff
@@ -38,6 +38,7 @@ public enum AutoModes {
         THREE_NOTE_65("ThreeNotesSourceSide65Choreo", StartPosition.STARTING_LOCATION_SOURCE_SIDE, List.of(6, 5)),
         //two note autos - Amp Side
         //two note autos - middle
+        THREE_NOTE_MIDDLE_52("ThreeNoteMiddle52Choreo", StartPosition.STARTING_LOCATION_MIDDLE, List.of(5,2)),
```
Comment (pjreiniger-aurora) [inline y2024/Crescendo/src/main/java/com/gos/crescendo2024/auton/Autos.java:41]: Put in 3 note section

File: y2024/Crescendo/src/main/java/com/gos/crescendo2024/auton/Autos.java [hunk 4]
```diff
@@ -50,6 +51,8 @@ public enum AutoModes {
         // No shooting
         NO_NOTE_LEAVE_WING("NoNoteLeaveWingChoreo", StartPosition.CURRENT_LOCATION, List.of()),
         //Disruption auto
+        ASSHOLEAUTO("AThreeNoteSource65Choreo", StartPosition.STARTING_LOCATION_SOURCE_SIDE, List.of()),
```
Comment (pjreiniger-aurora) [inline y2024/Crescendo/src/main/java/com/gos/crescendo2024/auton/Autos.java:54]: Put in 3 note section

File: y2024/Crescendo/src/main/java/com/gos/crescendo2024/RobotContainer.java (hunk 2 above)
Comment (pjreiniger-aurora) [reply y2024/Crescendo/src/main/java/com/gos/crescendo2024/RobotContainer.java:244]: This name change will break the webdash, so you will have to reconnect the command to the button with the appropriate name

**Review by pjreiniger — APPROVED** [review]

**Pull request merged**